.yamllint

```diff
@@ -17,8 +17,7 @@ rules:
   comments-indentation:
     level: "warning"
   document-end: "disable"
-  document-start:
-    present: true
+  document-start: "disable"
```

Comment:
why?
https://yaml.org/spec/1.1/#id897596

Reply:
`# ui_lovelace_minimalist` must be on the first line and can't be excluded with another comment. And it's a nice to have in yaml.

Reply:
:-/